src/it/resources/org/checkstyle/suppressionxpathfilter/parameterassignment/SuppressionXpathRegressionParameterAssignmentLambdas.java

```diff
@@ -0,0 +1,18 @@
+package org.checkstyle.suppressionxpathfilter.parameterassignment;
+
+public class SuppressionXpathRegressionParameterAssignmentLambdas {
+
+     public interface SomeInterface {
+        void method(int a);
+    }
+
```

Comment:
@romani , I had a doubt regarding Checkstyle's indentation checks.

In this file, the indentation of line 5 is generally matched with line 7.
I unknowingly added the extra whitespace character before "public" in line 5 but the checks have all passed.

I wanted to clarify with you if this is expected or not ? Should checkstyle be reporting this as violation or not ?

Reply:
@MANISH-K-07 our checks for test inputs are more liberal than our checks for production code. Think about it like this: we need to have weird/ugly input for checks to have something to complain about in tests :)